AsyncDisplayKit/ASDisplayNode.h

```diff
@@ -599,12 +569,35 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (CGRect)convertRect:(CGRect)rect fromNode:(nullable ASDisplayNode *)node;
 
+
+#pragma mark - Deprecated
```

Comment:
Optional: I've seen some frameworks move deprecated methods into a separate category to avoid listing them in the regular class interface. I think this pragma approach is fine but there are alternatives if that seems like a good idea.